images/macos/toolsets/toolset-12.json

```diff
@@ -15,19 +15,26 @@
             "versions": [ "2019", "2022" ]
         },
         "mono-versions": [
-            "6.12.0.174"
+            "6.12.0.179"
         ],
         "ios-versions": [
-            "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
+            "15.10.0.5", "15.8.0.3", "15.6.0.3", "15.4.0.0", "15.2.0.17", "15.0.0.6"
         ],
         "mac-versions": [
-            "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
+            "8.10.0.5", "8.8.0.3", "8.6.0.3", "8.4.0.0", "8.2.0.17", "7.14.0.27"
         ],
         "android-versions": [
-            "12.2.4.3", "12.1.0.2", "12.0.0.3", "11.3.0.4"
+            "12.2.8.3", "12.1.0.2", "12.0.0.3", "11.3.0.4"
         ],
         "bundle-default": "6_12_16",
```

Comment:
should not we bump the default bundle, does it work with current-default xcode?

Reply:
I changed Xcode & bundle-default versions